app/Resources/translations/messages.en.xlf

```diff
@@ -276,6 +276,14 @@
                 <source>post.deleted_successfully</source>
                 <target>Post deleted successfully!</target>
             </trans-unit>
+            <trans-unit id="post.comment_added">
```

Comment:
Just for consistency matter (with event name and method name) shouldn't be named `post.comment_created` instead?